src/exporter.js

```diff
@@ -116,6 +117,7 @@ export function handler (input, context, callback) {
                 outputCsvRow[DELIVERY_DATE] = deliveryDate
                 outputCsvRow[CHARGE_DAY] = chargeDay
                 outputCsvRow[CUSTOMER_PHONE] = data[WORK_PHONE]
+                outputCsvRow[ADDITIONAL_INFORMATION] = data[CONTACT_DESCRIPTION]
```

Comment:
yes the current fulfilment files we got from salesforce have the delivery instructions in the "additional information" column even though there are a bunch of more specific columns that are left empty like "where to leave" or "house details".